pkg/multiclustermanager/routes.go

```diff
@@ -111,9 +110,21 @@ func router(ctx context.Context, localClusterEnabled bool, tunnelAuthorizer *mcm
 	authed.PathPrefix("/v3/token").Handler(tokenAPI)
 	authed.PathPrefix("/v3").Handler(managementAPI)
 
+	// Metrics authenticated route
+	metricsAuthed := mux.NewRouter()
+	metricsAuthed.UseEncodedPath()
+	impersonatingAuth := auth.ToMiddleware(requests.NewImpersonatingAuth(sar.NewSubjectAccessReview(clusterManager)))
+	tokenReviewAuth := impersonatingAuth.Chain(auth.ToMiddleware(requests.NewTokenReviewAuth(scaledContext.K8sClient.AuthenticationV1())))
+	metricsAuthed.Use(mux.MiddlewareFunc(tokenReviewAuth))
+	metricsAuthed.Use(mux.MiddlewareFunc(rbac.NewAccessControlHandler()))
```

Comment:
Can we reuse the `rbac.NewAccessControlHandler()` from above instead of calling it independently?